src/segment-loader.js

```diff
@@ -1203,6 +1427,16 @@ export default class SegmentLoader extends videojs.EventTarget {
     this.pendingSegment_ = null;
     this.recordThroughput_(segmentInfo);
     this.addSegmentMetadataCue_(segmentInfo);
+    if (!this.playlist_.attributes.BANDWIDTH) {
+      if (!this.nextSegmentSize_) {
+        this.nextSegmentSize_ = segmentInfo.byteLength;
+      } else {
+        this.nextSegmentSize_ -= this.nextSegmentSize_ /
```

Comment:
wouldn't this just be `NaN` if `nextSegmentSize_` is null?

Reply:
Yeah that sounds right, in which case I think any numerical comparison would fail, yeah?